app/config/featureSwitches/FeatureSwitch.scala

```diff
@@ -22,13 +22,17 @@ sealed trait FeatureSwitch {
 
 object FeatureSwitch {
   val prefix: String = "feature.switch"
-  val listOfAllFeatureSwitches: List[FeatureSwitch] = List(ShowURBanner)
+  val listOfAllFeatureSwitches: List[FeatureSwitch] = List(ShowURBanner, AddedFAPIsNone)
 }
 
 case object ShowURBanner extends FeatureSwitch {
   override val name: String = s"${FeatureSwitch.prefix}.show-ur-banner"
 }
 
+case object AddedFAPIsNone extends FeatureSwitch {
```

Comment:
Could this feature switch name be made more generic? As it may relate to other changes we have planned.